lib/Braintree.php

```diff
@@ -139,8 +137,8 @@
 require_once('Braintree/Exception/TestOperationPerformedInProduction.php');
 require_once('Braintree/Test/Transaction.php');
 
-if (version_compare(PHP_VERSION, '5.4.0', '<')) {
-    throw new Braintree_Exception('PHP version >= 5.4.0 required');
+if (version_compare(PHP_VERSION, '5.3.0', '<')) {
+    throw new Braintree_Exception('PHP version >= 5.3.0 required');
```

Comment:
I didn't add this because it's only packaged for our internal use. I'll add to the package description to make it clearer for our future selves.